src/components/profile/profile.spec.js

```diff
@@ -1,5 +1,4 @@
 import React from "react";
-import TestUtils from "react-dom/test-utils";
 import { shallow, mount } from "enzyme";
 import TestRenderer from "react-test-renderer";
```

Comment:
I think this can be removed as the test that uses it seems redundant as setting the `large` prop doesn't seem to do anything as far as I can tell